backend/tournesol/views/exports.py

```diff
@@ -31,7 +31,7 @@ def write_logged_user_comparisons_file(request, write_target):
 
     Comparisons from all polls are included.
     """
-    fieldnames = ["video_a", "video_b", "criteria", "weight", "score"]
+    fieldnames = ["video_a", "video_b", "criteria", "weight", "score", "week_date"]
```

Comment:
This new column "week_date" is currently empty in the personal export. It should either be removed, or implemented below.

I don't think this has been specified yet. So I would remove it for now. And as there is no privacy concern in this personal dataset, so the exact datetime could be used here, in a future PR.

Reply:
done, thanks!